reset_db.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+echo 'drop database domjudge' | mysql -pdomjudge
```

Comment:
Don’t add this magic file 🙈

Reply:
I think we should... but maybe in a different PR.